integration_tests/tests/temporary-accommodation/manage/extension.cy.ts

```diff
@@ -14,6 +14,29 @@ context('Booking extension', () => {
     cy.task('stubAuthUser')
   })
 
+  it('navigates to the booking extension page', () => {
```

Comment:
This is a strong opinion, loosely held, and don't feel you need to change anything, but I feel like integration tests for small things, like making sure you can navigate to a page, would be better rolled into the main feature, as Cypress tests tend to come with a quite hefty price in terms of run time. It's entirely personal preference though, and I can see both sides of the argument.

Reply:
Hmm. My thinking was, do this bit of navigation once in it's own test, then I can be quick/lazy in the rest of the tests and just jump directly to the page. Though if there's a clear happy path to roll this into....